File: truss-compiler-maven-plugin/src/main/java/truss/compiler/generator/bound/xml/BoundProperty.java
package truss.compiler.generator.bound.xml;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlAttribute;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name = "property")
@XmlAccessorType(XmlAccessType.FIELD)
public class BoundProperty {
    @XmlAttribute
    private String name;
    @XmlAttribute
    private String type;
    @XmlAttribute
    private boolean list;
    @XmlAttribute
    private boolean nullable;
    @XmlAttribute
    private boolean last;

    public String getName() {
        return name;
    }

    public String getType() {
        return type;
    }

    public boolean isList() {
        return list;
    }

    public boolean isNullable() {
        return nullable;
    }

    public boolean isLast() {
        return last;
    }
}

File: Truss.Compiler.Generate/Bound/Xml/BoundProperty.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Xml.Serialization;

namespace Truss.Compiler.Generate.Bound.Xml {
    [XmlRoot("property")]
    public class BoundProperty {
        [XmlAttribute("name")]
        public string Name { get; private set; }

        [XmlAttribute("type")]
        public string Type { get; private set; }

        [XmlAttribute("list")]
        public bool IsList { get; private set; }

        [XmlAttribute("nullable")]
        public bool IsNullable { get; private set; }

        [XmlAttribute("last")]
        public bool IsLast { get; private set; }
    }
}
